ecc_enclave/enclave/shim.cpp

```diff
@@ -49,7 +49,10 @@ void get_state(const char* key, uint8_t* val, uint32_t max_val_len, uint32_t* va
         sgx_sha256_msg(val, *val_len, &state_hash);
```

Comment:
also, should we change the get_state to return an error?

Reply:
I agree, eventually yes! I create a issue for that